vendor/engines/split_accounts/app/assets/javascripts/split_account_splits.js.coffee

```diff
@@ -0,0 +1,17 @@
+# Update the `percent` value in each visible `split` so the total of all splits
+# adds up to 100.
+updateSplits = (event, param) ->
+  switch param.object_class
+    when "split", "split_accounts/split"
+      $container = $(event.target).closest("[data-subaccounts]")
+      $inputs = $container.find("[data-percent]").filter(":visible")
+
+      if $inputs.length > 0
+        percent = Math.round((100.0 / $inputs.length) * 100) / 100
+        remainder = Math.round((100.0 - percent * ($inputs.length - 1)) * 100) / 100
+        $inputs.val(percent)
+        $inputs.last().val(remainder)
+
+# Register event listeners when nested_form_fields are added or removed
+$(document).on "fields_added.nested_form_fields", updateSplits
```

Comment:
I know we're only including the JS on the single view, but what are your thoughts on the specificity of this? Is it possible to attach this only to the specific form (maybe with a `.js--` prefix?